Sources/Holocron/FileSystemStorageProvider.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  FileSystemStorageProvider.swift
+//  Holocron-iOS
+//
+//  Created by Pranjal Satija on 4/15/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
`CocoaPods` can be replaced with `Bottle Rocket Studios`
I've seen this in other file comments, so please fix those as well